_data/sw_clients.yml

```diff
@@ -71,6 +80,11 @@
           - server-time
           - monitor
           - userhost-in-names
+        SASL:
+          - dh-aes
+          - dh-blowfish
```

Comment:
Source for dh-aes and dh-blowfish being supported by HexChat? I see only `SASL (username + password)` and `SASL EXTERNAL (cert)` in the login methods and am quite sure that they were removed long time ago.